db/migrate/20201215215458_create_nod_date_update.rb

```diff
@@ -0,0 +1,13 @@
+class CreateNodDateUpdate < Caseflow::Migration
+  def change
+    create_table :nod_date_updates do |t|
+      t.references "appeal", null: false, foreign_key: true, comment: "Appeal for which the NOD date is being edited"
+      t.date "old_date", null: false, comment: "Date before update"
+      t.date "new_date", null: false, comment: "Date after update"
+      t.references "user", null: false, foreign_key: true, comment: "User that created this record"
```

Comment:
Let's add a comment to the table:
```suggestion
      t.references "user", null: false, foreign_key: true, comment: "User that updated the NOD date"
```